docs/topics_5/web-create-project.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='add_project_{context}']
+= Creating a project
+
+You create a project using a wizard-like interface.
+
+.Procedure
+
+. In the navigation pane, click *Projects*.
+. On the *Projects* page, click *Create project*.
++
+.Create project
+image::web-add-project_51.png[Create project]
+
+. Enter a unique name for the project.
+. Optional: Enter a description.
+. Click *Next*.
++
+.Add applications
+image::web-add-apps_51.png[Add applications]
++
+You can upload applications or register the server path that contains an application.
+
+. To upload applications:
+
```

Comment:
If you want to upload an application to the web console, click Browse, select the application file or files, and click Close.